Resources/views/Security/base_login.html.twig

```diff
@@ -83,7 +83,7 @@ file that was distributed with this source code.
         </div>
         <div class="col-sm-6">
             {% block sonata_user_registration %}
-                {% render controller("SonataUserBundle:RegistrationFOSUser1:register") %}
+                {% render controller("SonataUserBundle:RegistrationFOSUser2:register") %}
```

Comment:
@pulse00 should we have 2 templates here or at least a variable to use version 1 or version 2 ?